### DISCUSSION TITLE
点击版本升级后, 无意间发现菜单不在了

### DISCUSSION THREAD

**Discussion by gkerry** [1Panel-dev/1Panel#4610] [Q&A]
效果如下
![image](https://github.com/1Panel-dev/1Panel/assets/50052250/e77a7b7c-6b87-486f-8d03-1e9b9531b389)
参考官方demo, 少了 div.app-sidebar这个元素. 只剩下app-main的部分

不知道是bug. 还是我升级造成的. 救命啊
PS: 参考官方demo
![image](https://github.com/1Panel-dev/1Panel/assets/50052250/a10dea4d-5b07-4258-b665-6175b78fbebe)

**Comment by wanghe-fit2cloud** [comment]
清理浏览器缓存或者切换其他浏览器试一下。

**Comment by gkerry** [comment]
谢谢，切换用户就好了
